apis/openstack-keystone/src/main/java/org/jclouds/openstack/v2_0/functions/PresentWhenExtensionAnnotationNamespaceEqualsAnyNamespaceInExtensionsSet.java

```diff
@@ -42,6 +43,18 @@
 /**
  * We use the annotation {@link Extension} to bind a class that implements an extension
  * API to an {@link Extension}.
+ *
+ * Match in the following order:
+ *
+ * 1. Match by namespace
+ * 2. Match by namespace aliases
+ * 3. Match by alias
+ * 4. Match by name
+ *
+ * New versions of openstack have no namespaces anymore.
+ * Alias is different than a namespace alias - it's an alternative namespace URL to match against.
+ *
+ * TODO: Rename PresentWhenExtensionAnnotationNamespaceEqualsAnyNamespaceInExtensionsSet
```

Comment:
The comment indicates that you feel the name is no longer appropriate, and I agree. Usage count apart (the global rename can be mostly addresed with a search/replace), IMO it won't be a more convenient time to address this than now.